config.json

```diff
@@ -1,14 +1,14 @@
 {
-    "range": {
-        "low": "1",
-        "high": "8888"
-    },
-    "ipRange": {
-        "low": "0",
-        "high": "255"
-    },
-    "count": "10",
-    "thread": {
-        "count": "8"
-    }
-}
```

Comment:
Changes in `config.json` are not of any relevance.

Reply:
Was creating a merge conflict..So for resolving that i had to do this made not commits..in the previous commit I've added all the required data